programs/drift/src/instructions/user.rs

```diff
@@ -149,7 +149,8 @@ pub fn handle_initialize_user<'c: 'info, 'info>(
 
     let now_ts = Clock::get()?.unix_timestamp;
 
-    user.last_fuel_bonus_update_ts = now_ts;
+    user_stats.last_fuel_if_bonus_update_ts = now_ts.cast()?;
```

Comment:
should this only happen if user stats is zero? so it doesnt over write second time user stats is created? or it should just be in init user stats

Reply:
ah yeah it should probably just be on init user stats